util/rand_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package util
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestRandString(t *testing.T) {
+	s, err := RandomID("prefix=")
+	t.Logf("Rand string: %s", s)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !strings.HasPrefix(s, "prefix=") {
+		t.Errorf("Invalid prefix: %s", s)
+	}
+	if len(s)-len("prefix.") != 52 {
```

Comment:
ah.. the random id is always 256 bits (as suggested by https://gist.github.com/tqbf/be58d2d39690c3b366ad)... this means the padding will always be the same